src/benchmarklib/abstract_table_generator.hpp

```diff
@@ -64,6 +64,8 @@ class AbstractTableGenerator {
 
   static std::shared_ptr<BenchmarkConfig> create_benchmark_config_with_chunk_size(ChunkOffset chunk_size);
 
+  bool cold_start_used = false;
```

Comment:
would be really great to be able to pass this as an argument to the executable.
For our purposes now perhaps analogous to `_benchmark_config->cache_binary_tables` 
and later on something similarly for 
`./hyriseConsole --start-from-persisted-data <path_to_data>`